[web-pubsub-client-protobuf] Add more tests

zackliu opened the pull request:
### Packages impacted by this PR
eb-pubsub-client-protobuf

### Issues associated with this PR


### Describe the problem that is addressed by this PR
Add more tests

### What are the possible designs available to address the problem? If there are more than one possible design, why was the one in this PR chosen?


### Are there test cases added in this PR? _(If not, why?)_


### Provide a list of related PRs _(if any)_


### Command used to generate this PR:**_(Applicable only to SDK release request PRs)_

### Checklists
- [ ] Added impacted package name to the issue description
- [ ] Does this PR needs any fixes in the SDK Generator?** _(If so, create an Issue in the [Autorest/typescript](https://github.com/Azure/autorest.typescript) repository and link it here)_
- [ ] Added a changelog (if necessary)

mpodwysocki reviewed (changes requested):
Must address all comments in this PR before merging: https://github.com/Azure/azure-sdk-for-js/pull/24879

zackliu commented:
@mpodwysocki I think I've already addressed all the comments in the previous PR. The only thing is @xirzec mentioned we'd better remove `long`, but I've post some reason. Please check.

mpodwysocki commented:
@zackliu You have not resolved all comments from the previous PR #24879, including mine.  In order to merge this PR, all comments must be addressed in that PR.
